addons/website_event/data/event_demo.xml

```diff
@@ -139,8 +139,8 @@
 </div>
 
 
-<div class="container oe_dark">
-<div class="row">
+<div class="container">
+<div class="row oe_dark">
```

Comment:
If you set a background on a row, the color will overflow the container... so no.